README.md

```diff
@@ -37,40 +37,35 @@ The contents of the repository.
 | `config/mbed_cloud_client_user_config.h` | A configuration file for the Device Management Client settings.
 | `config/mbedtls_mbed_client_config.h`    | A configuration file for Mbed TLS.
 
-## Dependencies
+## Prerequisites
 
-Currently, there are a few dependencies in the build system:
-
-* librt
-* libstdc++
-
-Install these in Ubuntu 18.04:
-
-```
-$ apt install libc6-dev
-$ apt install libmosquitto-dev mosquitto-clients
-```
-
-## Build tool dependencies
-
-Tools needed for building:
- * Git for cloning this repository.
- * CMake 2.8 or later.
- * GCC for compiling.
- * Doxygen for documentation generation.
- * Graphviz for documentation generation.
+### 1. Install these in Ubuntu 18.04:
 
 ```
-$ apt install build-essential cmake git doxygen graphviz
+sudo apt install build-essential clang cmake curl doxygen gcc git graphviz libc6-dev libclang-dev libcurl4-openssl-dev libmosquitto-dev mosquitto-clients pkg-config python3 python3-pip
```

Comment:
python3-venv also needed, IMHO.

Reply:
I can add that now, but maybe we add virtualenv usage later